Manager/BackupManager.php

```diff
@@ -77,28 +77,13 @@ public function execute()
         try {
             // If we catch an exception or not, we would still like to try cleaning up after us
             $this->logger->info('[dizda-backup] Cleaning up after us.');
-            $this->processor->cleanUp();
+            $this->processorManager->cleanUp();
         } catch (IOException $e) {
-            $this->logger->error('[dizda-backup] Cleaning up failed.');
+            $this->logger->error('[dizda-backup] Cleaning up failed.', array('exception' => $e));
 
             return false;
         }
 
         return $successful;
     }
-
-    public function getClientManager()
-    {
-        return $this->cm;
-    }
-
-    public function getDatabaseManager()
-    {
-        return $this->dbm;
-    }
-
-    public function getProcessorManager()
```

Comment:
I guess not, I thought they were some legacy noise. Let’s keep them until we know a better way of dealing with the situation that @pieter2627 describes. 


> On 9 nov. 2015, at 13:17, Jon Gotlin <notifications@github.com> wrote:
> 
> In Manager/BackupManager.php <https://github.com/dizda/CloudBackupBundle/pull/88#discussion_r44268595>:
> 
> >  
> >              return false;
> >          }
> >  
> >          return $successful;
> >      }
> > -
> > -    public function getClientManager()
> > -    {
> > -        return $this->cm;
> > -    }
> > -
> > -    public function getDatabaseManager()
> > -    {
> > -        return $this->dbm;
> > -    }
> > -
> > -    public function getProcessorManager()
> So, should we really mark them as deprecated then?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/dizda/CloudBackupBundle/pull/88/files#r44268595>.
>